editor/store/effects.js

```diff
@@ -326,6 +326,11 @@ export default {
 		const { dispatch } = store;
 
 		let result;
+		// TODO: these are potentially undefined, this fix is in place
+		// until there is a filter to not use reusable blocks if undefined
+		if ( ! has( wp, 'api.models.Blocks' ) && ! has( wp, 'api.collections.Blocks' ) ) {
+			return;
+		}
```

Comment:
Let's move this check to the very top of the function. That way, the `let result` declaration on line 328 will be as near as possible to where it is used on line 334. This helps with code comprehension: readers won't have to guess if `result` will be mutated elsewhere in the function.

Reply:
👍